tests/pytorch/test_nn.py

```diff
@@ -1519,16 +1520,17 @@ def test_hgt(idtype, in_size, num_heads):
     train_idx = th.randint(0, 100, (10, ), dtype = idtype)
     sampler = dgl.dataloading.NeighborSampler([-1])
     train_loader = dgl.dataloading.DataLoader(g, train_idx.to(dev), sampler,
-                                            batch_size=8, device=dev,
+                                            batch_size=batch_size, device=dev,
                                             shuffle=True)
     (input_nodes, output_nodes, block) = next(iter(train_loader))
     block = block[0]
     x = x[input_nodes.to(th.long)]
     ntype = ntype[input_nodes.to(th.long)]
     edge = block.edata[dgl.EID]
     etype = etype[edge.to(th.long)]
-    y = m(block, x, ntype, etype)
-    assert y.shape == (block.number_of_dst_nodes(), head_size * num_heads)
+    if block.number_of_dst_nodes() == batch_size:
+        y = m(block, x, ntype, etype)
```

Comment:
But it means the sampled block cannot be used in training sometimes, right? If so, I believe there is a bug in the sampling algorithm.

Reply:
@BarclayII Any thoughts?

Reply:
@BarclayII Any thoughts?

Reply:
I don't think this is the correct fix.  See #4947.